packages/graphql/tests/tck/directives/vector/phrase.test.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Neo4jGraphQL } from "../../../../src";
+import { formatCypher, formatParams, translateQuery } from "../../utils/tck-test-utils";
+
+const queryName = "moviesVectorQuery";
+
+describe("phrase input - genAI plugin", () => {
+    let typeDefs: string;
+    let neoSchema: Neo4jGraphQL;
+    let verifyTCK;
```

Comment:
This isn't directly possible unfortunately as the type of `process.env.VERIFY_TCK` is a string. Also this would probably need to be changed holistically I think?

Perhaps we can look at this after this PR?

Reply:
that means that `VERIFY_TCK=false` would be treated as true which is not ideal.

Yeah, probably requires checking it everywhere

Reply:
I'll follow this up